git-d8ci

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
It's better to keep `/bin/sh` as the interpreter. This is a UNIX standard and is widely supported. If this stays sh and you have bash installed on your system, chances are that bash will be used.

On the other hand, if you prefer another implementation such as zsh, dash, tcsh, fish, or any other of the dozens of shells available, then having '/bin/bash' here will be annoying.